cluster/pre-existing/util.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
Is there any reason why you've placed this file here? Given that it defines kubemark's cloud provider interface (the one in test/kubemark/skeleton), test/kubemark/pre-existing/util.sh seems to be the right path to put this.